src/main/resources/assets/ae2stuff/lang/en_US.lang

```diff
@@ -41,11 +43,17 @@ ae2stuff.wireless.advtool.connector.next=Next connector in queue: %s,%s,%s
 ae2stuff.wireless.advtool.extra=§oSneak right click to change mode between queuing and binding mode
 ae2stuff.wireless.advtool.noconnectors=No connectors found in queue. Try putting the tool into queueing mode and adding some
 ae2stuff.wireless.advtool.pop=Removed %s,%s,%s from queue
+ae2stuff.wireless.advtool.hub.queued=%s hub slots added to queue!
+ae2stuff.wireless.advtool.queueing.clear=Queue is cleared!
+ae2stuff.wireless.tooltips.advtool.queueing.clear=Shift + Ctrl + R-click for clear queue!
+ae2stuff.wireless.tooltips.advtool.hubqols.binding=Ctrl + R-click for connect to all free hub slots!
+ae2stuff.wireless.tooltips.advtool.hubqols.queueing=Ctrl + R-click for add all free hub slots to queue!
```

Comment:
I would remove the exclamation marks from each of these entries (7 in total)

Reply:
done